src/v3/src/types/component.ts

```diff
@@ -24,6 +24,8 @@ export type TesterFunction = (
   uiElement: UISchemaElement | FieldElement | ButtonElement
 ) => boolean;
 
+export type HandleBlurFunction = (value: string | number | boolean, event: FocusEvent) => void;
```

Comment:
Focus change handler should be moved to the `handlers.ts` file as well. See the `ClickHandler` definition that already exists. The `ChangeHandler` is actually unused and also incorrectly typed, so we could remove it while you're in there.